var/spack/repos/builtin/packages/gpi-2/package.py

```diff
@@ -60,7 +60,8 @@ class Gpi2(AutotoolsPackage):
     depends_on('libtool',  type='build', when='@1.4.0:')
     depends_on('m4',       type='build', when='@1.4.0:')
 
-    depends_on('gawk', type='run')
+    depends_on('sed', type=('build','run'))
```

Comment:
Based on the PR description, it sounds like what you want here is something like:

```suggestion
    depends_on('sed', when='@:1.3.3', type=('build','run'))
```